vita3k/gui/src/common_dialog.cpp

```diff
@@ -104,7 +105,7 @@ static void draw_message_dialog(DialogState &common_dialog, float FONT_SCALE, Im
         ImGui::SetCursorPos(ImVec2(WINDOW_SIZE.x / 2 - buttons_width, WINDOW_SIZE.y - 50 * SCALE.y));
         ImGui::BeginGroup();
         for (int i = 0; i < common_dialog.msg.btn_num; i++) {
-            if (ImGui::Button(common_dialog.msg.btn[i].c_str(), BUTTON_SIZE)) {
+            if (ImGui::Button(common_dialog.msg.btn[i].c_str(), BUTTON_SIZE) || ImGui::IsKeyPressed(static_cast<ImGuiKey>(emuenv.cfg.keyboard_button_cross))) {
```

Comment:
About this, it won't work if there are multiple buttons being showed, it will activate for all of them at the same time, leading to only the last button being registered.
For example with this change, pressing the cross button on a dialog with options Ok and Cancel will cause the cancel button to be pressed

Reply:
i revert this